binapi/abf/abf.ba.go

```diff
@@ -292,6 +292,7 @@ func (m *AbfPolicyAddDel) Size() (size int) {
 		var s2 fib_types.FibPath
 		_ = s2
 		if j2 < len(m.Policy.Paths) {
+			//lint:ignore SA4006 we might not use this variable
```

Comment:
This should not be necessary if we get the top of file comment right: https://pkg.go.dev/cmd/go#hdr-Generate_Go_files_by_processing_source

If we lead with:
```
^// Code generated .* DO NOT EDIT\.$
```
well behaved linters (which includes everything I've ever run through golangci-lint) will ignore the file as generated.  Please note: I'm still in favor of *generating* as clean as we can, but linter comments to ignore a particular part of the file rather than the whole thing seems... perhaps a bit to specific.

Reply:
Agreed. Although I'm not in favor of ignoring generated code (it's still code that should be clean 😄). But agreed, we can solve this afterwords